BackEnd/src/routes/api/subject.js

```diff
@@ -22,7 +22,7 @@ createRoutes([
                 return res.status(400).json( {error: err.message} );
             }
 
-            const id = args.name.substring(0, 2).toUpperCase();
+            const id = Math.floor(Math.random()*15+5);
```

Comment:
Hier werden auch Ids gewürfelt?

Reply:
Siehe [oben](https://github.com/Freywood/GradeMe/pull/62#discussion_r512622095). Wird in #59 umgesetzt.